move_group/src/default_capabilities/move_action_capability.cpp

```diff
@@ -61,7 +61,6 @@ void move_group::MoveGroupMoveAction::initialize()
 void move_group::MoveGroupMoveAction::executeMoveCallback(const moveit_msgs::MoveGroupGoalConstPtr& goal)
 {
   setMoveState(PLANNING);
-  context_->planning_scene_monitor_->updateFrameTransforms();
```

Comment:
As far as I can see, this is still required.
The reason this is in there is that the `MoveGroupGoal` might contain a PlanningScene in `goal->planning_options.planning_scene_diff` and attached objects in the `RobotState` components.
These are given w.r.t. frames that might have to be transformed.
However, frame updates are done in all ***but*** the PlanningScene callback (for performance reasons).
Thus I don't see how frame updates are taken care of by `syncSceneUpdates` (which I suspect is the reason you removed it).